.rubocop.yml

```diff
@@ -25,6 +25,16 @@ AllCops:
   Exclude:
   - vendor/**/*
 
-# Disable temporarily
 Style/HashSyntax:
+  Description: Disable temporarily (FIXME)
+  EnforcedStyle: ruby19
   Enabled: false
+
+Naming/FileName:
+  Description: Some files violates the snake_case convention
+  Exclude:
+    - 'lib/beaker-docker.rb'
+
+# Style enforcements
+Style/TrailingCommaInArrayLiteral:
+  EnforcedStyleForMultiline: comma
```

Comment:
Any reason to prefer this over `consistent_comma`?

Reply:
Ah.. `consistent_comma` is what I actually want.. will fix it